### PR TITLE
Feat/13 icon component

### LINKED ISSUE #13: Icon component
Put together an icon component that uses remix icons and allows drop-in styling.

### PR DESCRIPTION
**Pull request by xanthipe**
**Issues affected**

Closes #13 

**Description**

* Adds the icon component
* Updates the Footer story to use autodocs
* Tweaks the build config to be more selective

**How to test**

Run up Storybook, check out the Icons and Footer components.

**Screenshots**

image
image

**Any other information?**

**Checklist**

- [x] Have all new components been added to Storybook?
- [ ] Have unit/component/integration tests been written?
- [x] Has documentation been updated?

### CODE REVIEW

File: public/favicon.png:Zone.Identifier
Comment (DiracLotus) [inline public/favicon.png:Zone.Identifier:1]: should this be committed?

File: src/components/display/icon.tsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import { FunctionComponent } from 'react';
+
+import { ColorTypes } from '../../themes/theme.types';
+
+import {
+  iconFillMap, IconName, iconsMap, iconStrokeMap,
+} from './icon.types';
+
+interface IconProps {
+  iconName: IconName;
```
Comment (DiracLotus) [inline src/components/display/icon.tsx:10]: do we want to be able to control the role from outside, for potential accessibility purposes?

Comment (xanthipe) [reply src/components/display/icon.tsx:10]: Yes, but if we do we'll want to be very careful about not using it for evil. Will add.

File: public/favicon.png:Zone.Identifier
Comment (xanthipe) [reply public/favicon.png:Zone.Identifier:1]: Nope! Removed!

**Review by DiracLotus — APPROVED** [review]

**Pull request merged**